examples/Makefile

```diff
@@ -0,0 +1,14 @@
+WORK_DIR:=$(strip $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+PROTO_HOME=/usr/local/opt/include
```

Comment:
Questionable portability

Reply:
open to suggestions

Reply:
At least add a comment what is supposed to be installed in /usr/local/opt/include, proto-something?